validation/src/test/scala/hmda/validation/engine/ts/quality/TsQualityEngineSpec.scala

```diff
@@ -24,7 +28,7 @@ class TsQualityEngineSpec
     val ts = line.map(l => TsCsvParser(l))
 
     ts.foreach { ts =>
-      checkQuality(ts.right.get, ctx).isSuccess mustBe true
+      checkQuality(ts.right.get, ctx).map(validation => validation.isSuccess mustBe true)
```

Comment:
Since this assertion is in the result of a future, it never gets tested. I think you'll have to change this spec to use `AsyncWordSpec`